lib/Test/PerlTidy.pm

```diff
@@ -129,6 +129,9 @@ sub list_files {
 
     my $finder = File::Finder->type('f')->name(qr{[.](?:pl|pm|PL|t)$});
     $finder->{options}->{untaint} = 1;
+    $finder->{options}->{untaint_pattern} =
+      qr{^((\p{IsAlphabetic}:)?[-+@\w./~[(][)] ]+)$}x ## no critic (Variables::ProhibitPunctuationVars)
```

Comment:
`Perl::Critic` misidentifies the `$}` at the end of the regex as a variable, so ignoring.